src/main/java/VASSAL/configure/ConfigureTree.java

```diff
@@ -284,6 +321,205 @@ protected JPopupMenu buildPopupMenu(final Configurable target) {
     addAction(popup, buildImportAction(target));
     return popup;
   }
+  
+  
+  // Enumerates our configure tree in preparation for searching it
+  private final List<DefaultMutableTreeNode> getSearchNodes(DefaultMutableTreeNode root) {
+    List<DefaultMutableTreeNode> searchNodes = new ArrayList<DefaultMutableTreeNode>();
```

Comment:
Can use the "diamond operator" `<>` here, the compiler is smart enough to figure out what type we need:
```
List<SomeType> myList = new ArrayList<>();
```